models.py

```diff
@@ -149,6 +149,26 @@ def deserialize(self, data, initialize_property=True):
                 self.restock_amt = DEFALUT_RESTOCK_AMT
 
         return self
+
+    def deserialize_restock(self, data, initialize_property=True):
+        """
+        Deserializes a ProductInformation from a dictionary.
+
+        Only restock_amt should exist in data.
+        """
+        if (not isinstance(data, dict)):
+            raise DataValidationError(BAD_DATA_MSG)
+
+        add_amt = data.get(RESTOCK_AMT)
+        if (len(list(data.keys())) != 1) or (add_amt is None) or (add_amt < DEFALUT_RESTOCK_AMT):
```

Comment:
Maybe we can use (add_amt < 0) instead of using DEFAULT_RESTOCK_AMT? because logically we only need to make sure the the restock_amt is positive and it doesn't matter if restock_amt is less or greater than the default value. It's just in our case that the default value happens to be 0, thus using default value instead of 0 might seem a little confusing.

Reply:
You are right. The same issue also goes for other attributes in deserialize() and deserialize_update().
@SaniyaLalani Just letting you know, I've changed some lines in deserialize_update(). You may want to take a look at my newest commit.